src/apis/alerts/index.ts

```diff
@@ -255,6 +394,9 @@ export class AlertsApiBridge {
     return this.bridge.postMessage(message);
   }
 
+  /**
+   * @deprecated This method is deprecated. Use getQueriesAlertsV2 instead.
```

Comment:
Wondering how we should handle deprecations for real Foundry users (customers). This notice _might_ show up for developers in their IDE in the best possible case (e.g. using TypeScript), but there is no guarantee. We could also call `console.warn()` here, to make this actually visible, at least in the browser console!

Also wondering if there is a plan how to actually remove these deprecated features, which would be a breaking change for customers. If people are loading the most current SDK from our servers, we don't want apps to suddenly stop working. Just mentioning this here, we can have further discussions elsewhere if needed I guess! /cc @RuslanZavacky

Reply:
@simonihmig if these methods are removed we'd release a new major version.  People have to load a specific version of the sdk in their code.  

Happy to chat more about this!